packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,158 +24,217 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-border-radius-xs * 2;
 
-  @include bpk-border-radius-pill;
+  svg {
+    width: 16 * $bpk-one-pixel-rem !important;
+    height: 16 * $bpk-one-pixel-rem !important;
+  }
+
```

Comment:
The icon size on figma is 16*16. Icons have an inline style attribute setting the height and width, so the only way to override this in CSS is to use !important.